code/game/atoms.dm

```diff
@@ -1319,6 +1319,8 @@
 	SEND_SIGNAL(src, COMSIG_ATOM_ENTERED, AM, oldLoc)
 	SEND_SIGNAL(AM, COMSIG_ATOM_ENTERING, src, oldLoc)
 
+	SEND_SIGNAL(src, COMSIG_MOVABLE_CROSSED, AM, oldLoc)
```

Comment:
I think you explained this to me, but remind me again why this is better than just using `_ATOM_ENTERED`? It appears to be identical.

Reply:
now mass replaced with entered (and uncrossed with exited)